src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -1036,6 +1036,64 @@ OverlayImpl::processRequest (http_request_type const& req,
     return true;
 }
 
+bool
+OverlayImpl::processValidatorList (http_request_type const& req,
+    Handoff& handoff)
+{
+    // If the target is in the form "/vl/<validator_list_public_key>",
+    // return the most recent validator list for that key.
+    // "unl" can be used as an alias for "vl", but this usage is discouraged
+    if ((!req.target().starts_with("/vl/") &&
+            !req.target().starts_with("/unl/")) ||
```

Comment:
I wouldn't allow `/unl/` at all. No point to provide a moniker but "discourage" it's use (i.e. make it "deprecated") in the initial version of an API.

Reply:
Good point. Fixed.